workspaces/ui-v2/src/optic-components/pages/diffs/ReviewEndpointDiffPage/ReviewEndpointDiffPage.tsx

```diff
@@ -106,6 +103,11 @@ export const ReviewEndpointDiffPage: FC<ReviewEndpointDiffPageProps> = ({
           <EndpointDocumentationPane
             method={method}
             pathId={pathId}
+            highlightedLocation={
```

Comment:
I realized I had a regression here when I split up this component - it now is fixed on the ReviewEndpointDiffPage... It should look something like this:
<img width="1203" alt="Screen Shot 2021-04-30 at 12 41 28 PM" src="https://user-images.githubusercontent.com/18374483/116746520-714b5500-a9b1-11eb-8055-812f1c69c830.png">